docs/workforce-management-framework/topics/pro-starting-the-core-server.adoc

```diff
@@ -0,0 +1,17 @@
+[id='pro-starting-the-core-server-{chapter}']
+= Starting the Core Server
+
+.Prerequisites
+
+. xref:pro-installing-core-dependencies-{chapter}[Installing Core Dependencies]
+
+.Procedure
+
+== Start the Core Node.js Server
+
+[source,bash]
+----
+npm run start
+----
+
+NOTE: The Core server requires mongodb and redis to be running
```

Comment:
Should we be assisting the reader in achieving this? Reference other material for example?